### PR TITLE
feat: Adding Brading/Logos page

### LINKED ISSUE #878: Add a Branding/Logos Page
Recently [saw on Twitter](https://twitter.com/t3dotgg/status/1559278052550488064?s=20&t=HoAUadi43RfhI7cvaTc5mQ) praise for [Astro's branding page](https://astro.build/press/). We could probably do something similar, providing a page of logos and assets for others to use when referring to Preact.

Some other examples:
  - [Remix.Run](https://drive.google.com/drive/u/0/folders/1pbHnJqg8Y1ATs0Oi8gARH7wccJGv4I2c)
  - [Vercel](https://vercel.com/design)

Common practice seems to be redirecting to these assets upon right-clicking the logo on their respective sites.

I'll probably try to write up a PR for this over the next few days.

### PR DESCRIPTION
**Pull request by rschristian**
Closes #878 

Adds a page containing some of our logos in SVG & PNG formats for users to download.

The going practice (used by Astro, Vercel, Remix, and likely others) seems to be redirecting users to this page if they right click on logos, so that's what I matched.

### CODE REVIEW

**Review by JoviDeCroock — APPROVED** [review]
Nice!

**Review by marvinhagemeister — APPROVED** [review]
Ohh this is fantastic!! 💯

File: src/components/header/index.js [hunk 1]
```diff
@@ -163,8 +166,20 @@ const NavLink = ({ to, isOpen, route, ...props }) => {
...
+	}
+
 	return (
-		<a href={to.href || to.path} {...props} data-route={route}>
+		<a
+			href={to.href || to.path}
+			{...props}
+			data-route={route}
+			onContextMenu={BrandingRedirect}
+		>
```
[the remark this answers is not in the document]
Comment (rschristian) [reply src/components/header/index.js:182]: No, logo w/ text doesn't look very good.

Spacing is too tight and I don't want to muck around with creating a separate logo just for this.

**Comment by rschristian** [comment]
@developit You mentioned on Slack you might have more assets sitting around?

Additionally, I know you brought up the inverted not really being something we used besides on our own site, however, we probably need _some_ offering for dark themes. Do you have an alternative? The purple as a background doesn't provide enough contrast with dark themes.

**Review by marvinhagemeister — APPROVED** [review]
This is really great! I think we can always add more assets if needed after this PR 👍

**Comment by rschristian** [comment]
Hmm, switching to draft real quick because the Netlify preview isn't preventing the context menu, oddly.

**Pull request merged**